### ISSUE TITLE
nodejs.lua postInstall method has a NPE error.

### ISSUE THREAD

**Issue by yimiaoxiehou** [version-fox/version-fox-plugins#7]
```
Press ↑/↓ to select and press ←/→ to page, and press Enter to confirm
Installing nodejs@20.10.0...
Downloading... 100% [===============================================] (2.3 MB/s)         
Verifying checksum d3f0908a9d9190a8525c5b9a716ed91bb57e908555841b0c47f75b2a001ff91b...
Unpacking /root/.version-fox/cache/nodejs/node-v20.10.0-linux-x64.tar.gz...
Install nodejs@20.10.0 success! 
{"rootPath":"/root/.version-fox/cache/nodejs/v-20.10.0","sdkInfo":{"nodejs":{"name":"nodejs","path":"/root/.version-fox/cache/nodejs/v-20.10.0/nodejs-20.10.0","version":"20.10.0"}}}
/root/.version-fox/cache/nodejs/v-20.10.0
nil
plugin [PostInstall] method error: <string>:126: attempt to index a non-table object(nil) with key 'path'
stack traceback:
	<string>:126: in main chunk
	[G]: ?

```

**Comment by yimiaoxiehou** [comment]
the nodejs sdkInfo has nodejs prop not node prop
```json
{"rootPath":"/root/.version-fox/cache/nodejs/v-20.10.0","sdkInfo":{"nodejs":{"name":"nodejs","path":"/root/.version-fox/cache/nodejs/v-20.10.0/nodejs-20.10.0","version":"20.10.0"}}}
```

**Comment by aooohan** [comment]
Fixed by #8
